website/docs/UserEvent.md

```diff
@@ -28,3 +28,39 @@ Creates User Event instances which can be used to trigger events.
 ### Options
 - `delay` - controls the default delay between subsequent events, e.g. keystrokes, etc.
 - `advanceTimers` - time advancement utility function that should be used for fake timers. The default setup handles both real and Jest fake timers.
+
+
+## `press()`
+
+```ts
+type(
+  element: ReactTestInstance,
+): Promise<void>
+```
+
+Example
+```ts
+const user = userEvent.setup();
+
+await user.press(touchable);
+```
+
+This helper simulates a press on any pressable element, e.g. Pressable, Text or TextInput. Unlike fireEvent.press which is a simpler API that will only call the onPress prop, this simumlates the entire press event in a more realistic way by reproducing what really happens when a user presses a pressable component. This means for instance for onPressIn and onPressOut props will also be invoked.  
+
+## `longPress()`
+
+```ts
+type(
+  element: ReactTestInstance,
+  options: { pressDuration: number } = { pressDuration: 500 }
```

Comment:
[nit] Perhaps simple `duration` would be enough, as "press" should be evident from the context.

```suggestion
  options: { duration: number } = { duration: 500 }
```

Reply:
Yes i agree, I changed the option name